### DISCUSSION TITLE
using pb as cms ?

### DISCUSSION THREAD

**Discussion by Matrex7** [pocketbase/pocketbase#5375] [Q&A]
so i googled alot of stuff , and i like the jamstack approach to making blog 
there is a "trick" i read about , when you add or change content say "post" collection , a hook triggers to for example github repo to trigger a build process , i think about  using js hooks to intercept the crud operations then doing the build . but im bit lost .
any help would be good

**Comment by jonshipman** [comment]
I don't think I follow. Pocketbase is whatever you want it to be.

The admin GUI can be thought of as phpMyAdmin for mysql. If you're talking about making the "/_" admin url a CMS then that's not remotely secure (about as secure as giving users access to phpmyadmin). 

You shouldn't have to build anything for pocketbase. Are you talking about hosting the sqlite database on github? Again, not possible as you need a server to run the pocketbase binary.

And if you have a host for pocketbase you can spin up a webserver as well.
